ui/src/components/WMWorkersTable/index.jsx

```diff
@@ -0,0 +1,257 @@
+import React, { Component } from 'react';
+import { withRouter } from 'react-router-dom';
+import { isEmpty, map, pipe, sort as rSort } from 'ramda';
+import { camelCase } from 'change-case';
+import memoize from 'fast-memoize';
+import { formatDistanceStrict } from 'date-fns';
+import { CopyToClipboard } from 'react-copy-to-clipboard';
+import ContentCopyIcon from 'mdi-react/ContentCopyIcon';
+import AlertIcon from 'mdi-react/AlertIcon';
+import LinkIcon from 'mdi-react/LinkIcon';
+import TableRow from '@material-ui/core/TableRow/TableRow';
+import TableCell from '@material-ui/core/TableCell/TableCell';
+import ListItemText from '@material-ui/core/ListItemText/ListItemText';
+import Typography from '@material-ui/core/Typography/Typography';
+import DataTable from '../DataTable';
+import TableCellListItem from '../TableCellListItem';
+import Link from '../../utils/Link';
+import DateDistance from '../DateDistance';
+import sort from '../../utils/sort';
+import { WMWorkers } from '../../utils/prop-types';
+
+@withRouter
+export default class WorkerManagerWorkersTable extends Component {
+  static propTypes = {
+    searchTerm: String,
+    workers: WMWorkers.isRequired,
+  };
```

Comment:
For non required `propTypes`, it would be nice to have `defaultProps` defined.

```jsx
static propTypes = {
 // ...
};

static defaultProps = {
  searchTerm: '',
};
```